kubectl-minio/cmd/tenant-create.go

```diff
@@ -142,11 +143,34 @@ func createTenant(oclient *operatorv1.Clientset, kclient *kubernetes.Clientset,
 	if err != nil {
 		return err
 	}
-	fmt.Printf("MinIO Tenant %s Created\n\n", to.ObjectMeta.Name)
-	fmt.Printf("Tenant\nAccess Key: %s\nSecret Key: %s\nVersion: %s\nClusterIP Service: %s\n\n", s.Data["accesskey"], s.Data["secretkey"], to.Spec.Image, to.Spec.ServiceName)
-	fmt.Printf("MinIO Console\nAccess Key: %s\nSecret Key: %s\nVersion: %s\nClusterIP Service: %s\n\n", console.Data["CONSOLE_ACCESS_KEY"], console.Data["CONSOLE_SECRET_KEY"], to.Spec.Console.Image, to.Spec.Console.Metadata.Name+miniov1.ConsoleName)
-	if t.Spec.KES != nil {
-		fmt.Printf("KES Version: %s\n", t.Spec.KES.Image)
+	if color.IsTerminal() {
+		consolePort := miniov1.ConsolePort
+		minioPort := miniov1.MinIOPort
+		if to.HasCertConfig() || to.AutoCert() {
+			consolePort = miniov1.ConsoleTLSPort
+		}
+		printBanner(to.ObjectMeta.Name, string(console.Data["CONSOLE_ACCESS_KEY"]), string(console.Data["CONSOLE_SECRET_KEY"]),
+			to.ConsoleCIServiceName(), to.MinIOHLServiceName(), consolePort, minioPort, (to.HasCertConfig() || to.AutoCert()))
 	}
 	return nil
 }
+
+func printBanner(tenantName, user, pwd, consoleSVCName, minioSVCName string, consolePort, minioPort int, tls bool) {
+	minioLocalPort := 9000
+	consoleLocalPort := 9090
+	scheme := "http"
+	if tls {
+		scheme = "https"
+	}
+	fmt.Printf(color.Bold(fmt.Sprintf("\nMinIO Tenant '%s' created\n\n", tenantName)))
+	fmt.Printf(color.Blue("Username: ") + color.Bold(fmt.Sprintf("%s \n", user)))
+	fmt.Printf(color.Blue("Password: ") + color.Bold(fmt.Sprintf("%s \n\n", pwd)))
+
+	fmt.Printf(color.Blue("Web interface access: \n"))
+	fmt.Printf(fmt.Sprintf("\t$ kubectl port-forward svc/%s %d:%d\n", consoleSVCName, consoleLocalPort, consolePort))
+	fmt.Printf(fmt.Sprintf("\tPoint browser to %s://localhost:%d\n\n", scheme, consoleLocalPort))
+
+	fmt.Printf(color.Blue("Object storage access: \n"))
+	fmt.Printf(fmt.Sprintf("\t$ kubectl port-forward svc/%s %d:%d\n", minioSVCName, minioLocalPort, minioPort))
```

Comment:
I noticed in the output that this uses the `-hl` service on port 9000 - does that also handle TLS traffic? I've had some longstanding confusion on the difference between the `tenant1-hl:9000` service and the `tenant1-minio:9443` service (which feels like it's explicitly supporting TLS via 9443?)

Reply:
`tenant1-hl:9000` is a mandatory (headless) service. It does allow TLS as well. I chose this because we are sure that this service will always exist (it is required by statefulset). 

`tenant1-minio:9443` is optional, and created only when passed via yaml (plugin always creates it though).